### ISSUE TITLE
Compatibility with Deployer 7.X

### ISSUE THREAD

**Issue by ioniks** [Setono/deployer-cron#28]
Hi,

Have you planned to make it compatible with Deployer 7.X ?

**Comment by loevgaard** [comment]
Yes, definitely, but I am unable to because of this issue: https://github.com/deployphp/deployer/issues/3485. Please voice your opinion in the issue :)

Until then we are stuck on v6 :(

**Comment by ioniks** [comment]
You can't make a temporally patch with this solution?

https://github.com/deployphp/deployer/issues/3485#issuecomment-1482596556

**Comment by loevgaard** [comment]
As you can see from my comments it doesn't work

**Comment by ioniks** [comment]
I missed it 😭

**Comment by loevgaard** [comment]
I just released an 1.0.0@alpha that works in my tests that is compatible with v7. Maybe you could test it out?
